tests/TestHarness/Cluster.py

```diff
@@ -461,6 +461,11 @@ def connectGroup(group, producerNodes, bridgeNodes) :
                 if "--plugin eosio::history_api_plugin" in args:
                     argsArr.append("--is-nodeos-v2")
                     break
+
+        # Handle common case of specifying no block offset for older versions
+        if "v2" in self.nodeosVers or "v3" in self.nodeosVers or "v4" in self.nodeosVers:
```

Comment:
I like this a little better, no need to change though.
```suggestion
        if any(s in self.nodeosVers for s in ["v2", "v3", "v4"]):
```